src/components/Calculator.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import './Calculator.css';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Linting is important to reduce errors and improve the overall quality of your code  so I think fixing them would be much better than disabling them